exporters/autoexport/metrics.go

```diff
@@ -148,6 +172,10 @@ func init() {
 
 		return readerWithServer{lis.Addr(), reader, &server}, nil
 	})
+
+	RegisterMetricProducer("prometheus", func(ctx context.Context) (metric.Producer, error) {
+		return prometheusbridge.NewMetricProducer(), nil
```

Comment:
@gouthamve is it going to be a problem for any of your use-cases that this relies on the global prometheus default gatherer?

Reply:
That is a good point. I added a fallbackfactory for that.